README.md

```diff
@@ -801,6 +810,169 @@ Since the `script.sh` ends with `echo $? > rc`, the backend will wait for the ex
 
 ... more to come ...
 
+# Runtime Attributes
+
+Runtime attributes are used to customize tasks. Within a task one can specify runtime attributes to customize the environment for the call.
+
+For example:
+
+```
+task googly_task {
+  command {
+    echo "Hello JES!"
+  }
+  runtime {
+    docker: "ubuntu:latest"
+    memory: "4G"
+    cpu: "3"
+    defaultZones: "US_Metro US_Backwater"
+    defaultDisks: "Disk1 3 SSD, Disk2 500 OldSpinnyKind"
+  }
+}
+workflow googly_workflow {
+  call googly_task
+}
+```
```

Comment:
I wrote the above jokey WDL as part of a test, but I really didn't intend it as an exemplar for use in documentation.  :smile:  With just a few changes I think this would be okay: googly -> JES, US_Backwater -> MX_Metro, OldSpinnyKind -> HDD